### ISSUE TITLE
default database schema?

### ISSUE THREAD

**Issue by edisonrex** [vortexsystem/VortexWeb#12]
There is no schema for the default database? I recognise ci_sessions as a codeignitor session table, but how would a person who doesn't code, know this?

**Comment by Usalabs** [comment]
Don't bother, this script is not being developed anymore, it's dead.  Last entry was in 2019

**Comment by jacobfear** [comment]
Yes, unfortunetly, I had to stop development due to my high school dual enrollment work taking up all my time. Maintainers are welcome to take over it if anyone want.s

**Comment by edisonrex** [comment]
seemed like a nice project, thanks. I did figure out the schema, it was in a sql file in the repo. 

Good luck with school.

**Comment by Usalabs** [comment]
@edisonrex
> seemed like a nice project, thanks. I did figure out the schema, it was in a sql file in the repo. Good luck with school.
> […](#)

Where did you see the default schema?  I only see 'Demo.sql' which is not a schema for setting up the interface, 'Demo.sql' is a fully populated database for demo purposes.

There's only the one sql file and 'demo.sql' aint the default.

**Comment by edisonrex** [comment]
I’d consider a fully populated database for demo purposes to give me the database schema. All I need are the tables and the relations, which seem to be in there.
